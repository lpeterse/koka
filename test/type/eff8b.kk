fun forever(f) {
  f("hi");
  error("hi");
  forever(f);
}

fun test() {
  forever( print );
}