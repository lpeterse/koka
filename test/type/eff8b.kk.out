forever     : forall<a,b,e> ((string) -> <pure|e> a) -> <pure|e> b
test        : forall<a> () -> <pure,console> a
